src-electron/db/query-session.js

```diff
@@ -40,6 +40,65 @@ async function getAllSessions(db) {
   return rows.map(dbMapping.map.session)
 }
 
+/**
+ * Reloads a previous session.
+ *
+ * @export
+ * @param {*} db
+ * @param {*} sessionId
+ * @param {*} userRef
+ * @param {*} sessionKey
+ * @returns A promise that resolves with the number of rows updated.
+ */
+async function reloadSession(db, sessionId, userRef, sessionKey) {
+  return dbApi.dbUpdate(
+    db,
+    'UPDATE SESSION SET DIRTY = ?, USER_REF = ? , SESSION_KEY = ? WHERE SESSION_ID = ?',
+    [1, userRef, sessionKey, sessionId]
+  )
+}
+
+/**
+ * Returns a promise that resolves into an array of objects containing 'sessionId', 'sessionKey' and 'creationTime' and assigned packages.
```

Comment:
Function description should mention the dirty sessions and what that is for more clarity